src/components/Tooltip/docs/Tooltip.md

```diff
@@ -8,6 +8,23 @@ initialState = {
   style: 'primary',
   trigger: 'button',
   isVisible: false,
+  translate: 'none'
+}
+
+styles = {
```

Comment:
The translate docs are cool for testing, but I don't think this is helpful to have in the docs for users or designers, etc. Can we remove them?